source/transactions-convenient-api/transactions-convenient-api.md

```diff
@@ -128,11 +129,23 @@ This method should perform the following sequence of actions:
 6. If the callback reported an error:
     1. If the ClientSession is in the "starting transaction" or "transaction in progress" state, invoke
         [abortTransaction](../transactions/transactions.md#aborttransaction) on the session.
+
     2. If the callback's error includes a "TransientTransactionError" label and the elapsed time of `withTransaction` is
-        less than 120 seconds, jump back to step two.
+        less than 120 seconds, calculate the backoffMS to be
+        `jitter * min(BACKOFF_INITIAL * (1.25**retry), BACKOFF_MAX)` where:
+
+        1. jitter is a random float between \[0, 1)
+        2. retry is one less than the number of times Step 2 has been executed since Step 1 was executed
+        3. BACKOFF_INITIAL is 1ms
+        4. BACKOFF_MAX is 500ms
```

Comment:
The graphs and explanation you included in your presentation at the offsite were really helpful in understanding where these numbers came from.  What do you think of including them somewhere in the spec?  Most specs have the FAQ section at the bottom - we could add a FAQ that explains why we chose the values we did for BACKOFF_MAX and BACKOFF_INITIAL?